tests/http/cache-control.yml

```diff
@@ -0,0 +1,44 @@
+config: !file tests/http/config/simple.graphql
```

Comment:
[optional] You can checkout `!inline` and write the configuration inside the spec. That makes it easier to understand what's happening in the spec.

Reply:
I will checkout in next commit 👍🏻